specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/integrationRuntime.json

```diff
@@ -703,6 +703,12 @@
               "$ref": "#/definitions/IntegrationRuntimeListResponse"
             }
           },
+            "400": {
+              "description": "Bad Request",
+              "schema": {
+                "$ref": "#/definitions/IntegrationRuntimeListResponse"
+              }
+            },
```

Comment:
Why add an explicit "400" response when you already have "default"? We generally discourage defining explicit error codes and use "default" for all error cases.

Furthermore, I doubt that the response schema is correct here -- it is the same as the success response which is inconsistent with a "Bad Request".